knowledge-base/how-to-upgrade-an-old-project-to-the-latest-telerik-reporting-version.md

```diff
@@ -29,28 +29,28 @@ To upgrade Telerik Reporting to the latest version without a problem you can rel
 ## Solution  
 The recommended approach for upgrading is to download an official release of each year between the project's Telerik Reporting version and the target version. Then you can upgrade the project, and check each report in order to remove warning for obsolete code. A small change in the designer will cause it to reserialize the report definition.  
 
-Details how to download MSI files and upgrade project's items can be found in <a href="http://docs.telerik.com/reporting/installation-installing-from-msi" target="_blank">Installing Telerik Reporting</a> and <a href="http://docs.telerik.com/reporting/installation-upgrading-newer-version" target="_blank">Upgrading Telerik Reporting Projects</a>.  
+Details how to download MSI files and upgrade project's items can be found in [Installing Telerik Reporting](../installation) and [Upgrading Telerik Reporting Projects](../installation-upgrading-newer-version).  
 
 ## Notes
 Here are the most important changes in Telerik Reporting during the last several years and the respective information sources: 
 
-- Reports should be wrapped in a <a href="https://docs.telerik.com/reporting/report-sources" target="_blank">report source object</a> in order to be passed to a report viewer, SubReport or action - <a href="/support/kb/reporting/details/how-to-migrate-your-project-to-utilize-the-new-reportsource-objects" target="_blank">How to migrate your project to utilize the new ReportSource objects</a>.
+- Reports should be wrapped in a [report source object](../report-sources) in order to be passed to a report viewer, SubReport or action - [How to migrate your project to utilize the new ReportSource objects](./how-to-migrate-your-project-to-utilize-the-new-reportsource-objects).

```

Comment:
the link for report source object should be ../designing-reports/report-sources/overview